soc/arm/st_stm32/stm32g0/soc.h

```diff
@@ -44,6 +44,12 @@
 #include <stm32g0xx_ll_i2c.h>
 #endif
 
+#if defined(CONFIG_COUNTER_RTC_STM32)
+#include <stm32g0xx_ll_rtc.h>
+#include <stm32g0xx_ll_exti.h>
+#include <stm32g0xx_ll_pwr.h>
```

Comment:
Is pwr ll required here (or should be moved to CONFIG_CLOCK_CONTROL_STM32_CUBE) ?

Reply:
yes, moved to the CONFIG_CLOCK_CONTROL_STM32_CUBE section

Reply:
yes, moved to the CONFIG_CLOCK_CONTROL_STM32_CUBE section